### PR TITLE
add BottomDrawer component

### PR DESCRIPTION
**Pull request by kdembler**
This adds a reusable `BottomDrawer` component that can be used for any kind of action overlay that slides from the bottom. This will include:
- Video workspace
- NFT workspace
- NFT purchase view
- Settling an auction
- Anything similar

The code is mostly copied from the video workspace, with some adjustments. The component includes support for:
- Entering/exiting the screen with animation and background overlay
- Managing scroll lock
- Drawer header
- Optional action bar
- Scrollable content inside the overlay, with space reserved for action bar (no need to measure manually)
- Updated tab title when the overlay is opened

You can test it out with the `BottomDrawer` story. I haven't used it in the video workspace yet, since it needs to be slightly rebuilt first, will do that later

### CODE REVIEW

File: packages/atlas/src/components/_overlays/BottomDrawer/BottomDrawer.styles.ts [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+    }
+  }
+`
+
+export const Container = styled.div`
+  transform: translateY(100%);
+  opacity: 0;
+  position: fixed;
+  z-index: ${zIndex.videoWorkspaceOverlay};
+  top: var(--size-topbar-height);
```
Comment (rafalpawlow) [inline packages/atlas/src/components/_overlays/BottomDrawer/BottomDrawer.styles.ts]: this value cannot be fixed, e.g. in nft purchase drawer takes up the whole screen

File: packages/atlas/src/components/_overlays/BottomDrawer/BottomDrawer.styles.ts [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+    &-exit-active {
+      transform: translateY(100%);
+      opacity: 0;
+    }
+  }
+`
+
+type ScrollContainerProps = {
+  actionBarHeight?: number
+  isEdit?: boolean
```
Comment (rafalpawlow) [inline packages/atlas/src/components/_overlays/BottomDrawer/BottomDrawer.styles.ts]: I think it's unnecessary, we don't use it anywhere

**Review by rafalpawlow — CHANGES_REQUESTED** [review]

File: packages/atlas/src/components/_overlays/BottomDrawer/BottomDrawer.styles.ts (hunk 1 above)
Comment (kdembler) [reply packages/atlas/src/components/_overlays/BottomDrawer/BottomDrawer.styles.ts]: 👍 I missed that. Added a `coverTopbar` prop that if true will make the drawer fill the full screen height

File: packages/atlas/src/components/_overlays/BottomDrawer/BottomDrawer.styles.ts (hunk 2 above)
Comment (kdembler) [reply packages/atlas/src/components/_overlays/BottomDrawer/BottomDrawer.styles.ts]: Removed

**Review by rafalpawlow — APPROVED** [review]

**Pull request merged**